layers/meta-opentrons/recipes-robot/opentrons-module-firmware/opentrons-module-firmware.bb

```diff
@@ -5,21 +5,21 @@ LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/Apache-2.0;md5=89aea4e17d99a7ca
 
 BASE_URL="https://opentrons-modules-builds.s3.us-east-2.amazonaws.com"
 
-TEMPERATURE_MODULE_VERSION="v2.1.0"
+TEMPERATURE_MODULE_VERSION="v0.1.2"
 TEMPERATURE_MODULE_SOURCE="temp-deck-arduino.ino.hex"
 TEMPERATURE_MODULE_FILENAME="temperature-module@${TEMPERATURE_MODULE_VERSION}.hex"
 TEMPERATURE_MODULE_URL="temperature-module/${TEMPERATURE_MODULE_VERSION}/${TEMPERATURE_MODULE_SOURCE}"
-SRC_URI += "${BASE_URL}/${TEMPERATURE_MODULE_URL};sha256sum=4e4ef00b44a320fe461265dbd0484d90247a13f405125657be8915c8ea214408"
+SRC_URI += "${BASE_URL}/${TEMPERATURE_MODULE_URL};sha256sum=a5fc5749b8e71db5afb2ca4da46d023e2751f4a9c48dbecc2d2a9e1d85b984d3"
 
-THERMOCYCLER-GEN2_MODULE_VERSION="v1.0.6"
+THERMOCYCLER-GEN2_MODULE_VERSION="v1.0.8"
```

Comment:
should be v1.0.7?
https://github.com/Opentrons/opentrons-modules/releases/tag/thermocycler-gen2%40v1.0.7